guides/source/security.md

```diff
@@ -29,6 +30,147 @@ The threats against web applications include user account hijacking, bypass of a
 
 In order to develop secure web applications you have to keep up to date on all layers and know your enemies. To keep up to date subscribe to security mailing lists, read security blogs, and make updating and security checks a habit (check the [Additional Resources](#additional-resources) chapter). It is done manually because that's how you find the nasty logical security problems.
```

Comment:
I think this should be rephrased to this:

To develop secure web applications, you need to stay updated on all layers and understand potential threats. Stay informed by subscribing to security mailing lists, reading security blogs, and making regular updates and security checks part of your routine (refer to the [Additional Resources](#additional-resources) chapter). Manual checks are essential for uncovering tricky logical security issues.

Reply:
@Youxaf-khan certainly, thanks for the review. We will focus on the new Authentication section for this PR but the that wording will get an update later on.